src/rpc/blockchain.cpp

```diff
@@ -2211,6 +2215,8 @@ static RPCHelpMan savemempool()
 {
     const CTxMemPool& mempool = EnsureAnyMemPool(request.context);
 
+    auto node_context = util::AnyPtr<NodeContext>(request.context);
```

Comment:
Sorry, I mean the line with the EnsureNodeContext use...

Reply:
I'm not sure which line you're referring to.

This code no longer calls `EnsureAnyNodeContext` because` EnsureAnyMemPool` already does,

Reply:
That's an internal implementation detail of `EnsureAnyMemPool`. We also generally avoid pointers where nullptr is unacceptable.

```suggestion
    NodeContext& node = EnsureAnyNodeContext(request.context);
```

Reply:
Done in 09c3395 . Thanks for suggestion.